papers/ring_vrf/overview.tex

```diff
@@ -26,141 +34,151 @@ \section{Protocol overview}
     returns either an output $\Out$ or else failure $\perp$.
 \end{itemize}
 
-A ring VRF like this could be instantiated using a NIZK for a relation like
-$$ \Setst{ \Out, \msg, \ring}{
+The most critical difference between a ring VRF and a VRF is that the verification of ring VRF does not require the public key of the signer to verify the signature and to output the evaluation value generated by $ \rVRF.\Eval $. Therefore, the algorithm $ \rVRF.\Verify $ of a correct and secure ring VRF protocol should verify the following relation with a signature $ \sigma $ intuitively:  
+
+
+$$ \Setst{ ((\sk, \pk),(\Out, \msg, \ring))}{
     \eprint{
-        (\pk,\sk) \leftarrow \KeyGen, \quad %   \textrm{ and }
+        (\pk,\sk) \leftarrow \rVRF.\KeyGen(\secparam), \quad %   \textrm{ and }
         \pk \in \ring, \quad
-        \Out = \PRF(\sk,\msg)
+        \Out = \rVRF.\Eval(\sk,\msg)
     }{
         \begin{aligned}
-        & (\pk,\sk) \leftarrow \KeyGen \\
+        & (\pk,\sk) \leftarrow \KeyGen(\secparam) \\
         & \pk \in \ring \\
-        & \Out = \PRF(\sk,\msg)
+        & \Out = \rVRF.\Eval(\sk,\msg)
```

Comment:
This is interesting.  I wrote PRF here because doing a PRF in a NIZK should always satisfy the pseudo-randomness definitions, so PRF gave a more intuitive picture.  It's messy however because we do not define pseudo-randomness in that way.

Any thoughts @AlistairStewart ?